### DISCUSSION TITLE
How can I display the total data consumed in GB?

### DISCUSSION THREAD

**Discussion by manubuhay** [Salamek/huawei-lte-api#172] [Q&A] [answered]
I managed to find the "reboot" function by just doing the "dir" function on all the methods a few months ago.

Now I want to create a script where if the download section of the "Current Connection" table in the "Home" tab is over 50GB, I am going to reboot the CPE router. Any help is greatly appreciated!

Whether it is on the "Home" or "Statistics" tab, it doesn't matter, as long as I can find the total data consumed, and if it exceeds 50GB, the script reboots the CPE router.

---Home Tab---
![image](https://user-images.githubusercontent.com/18477232/215339846-b17c8d40-29fe-4017-866f-ffb3d178f40c.png)
---Statistics Tab---
![image](https://user-images.githubusercontent.com/18477232/215339671-5f4515ad-1caf-4dae-b510-11769a137007.png)

**Comment by Salamek** [comment, marked as answer]
Hi check this project that is using `huawei-lte-api` to display usage stats: https://github.com/octave21/huawei-lte

I think that it uses `monitoring.month_statistics()`...
